scripts/turtles-dev.sh

```diff
@@ -127,15 +162,15 @@ install_local_providers_chart() {
 }
 
 # patch the removed pre-install hook
-pre_install_configuration
+#pre_install_configuration
 
-echo "Installing local Rancher Turtles chart for development..."
-install_local_rancher_turtles_chart
+# echo "Installing local Rancher Turtles chart for development..."
+# install_local_rancher_turtles_chart
 
-echo "Installing local Rancher Turtles Providers..."
-install_local_providers_chart
+# echo "Installing local Rancher Turtles Providers..."
+# install_local_providers_chart
 
-if [ "$USE_TILT_DEV" == "true" ]; then
-    echo "Using Tilt for development..."
-    tilt up
-fi
+# if [ "$USE_TILT_DEV" == "true" ]; then
+#     echo "Using Tilt for development..."
+#     tilt up
```

Comment:
Does that work fine with tilt? doesn't system chart controller override the deployment?